src/modules/market/components/market-properties/market-properties.jsx

```diff
@@ -105,6 +120,10 @@ MarketProperties.propTypes = {
   showAdditionalDetailsToggle: PropTypes.bool,
   showingDetails: PropTypes.bool,
   toggleDetails: PropTypes.func,
+  updateModal: PropTypes.func.isRequired,
```

Comment:
.isRequired before optional prop types.


just add these properties to /src/modules/market/containers/market-properties.js
and bam done! don't have to push down the hierarchy tree.